core/eval.go

```diff
@@ -863,95 +915,117 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+// evalCOMMAND evaluates COMMAND <subcommand> command based on subcommand
+// COUNT: return total count of commands in Dice.
+func evalCOMMAND(args []string) []byte {
+	if len(args) == 0 {
+		return Encode(fmt.Errorf("(error) ERR wrong number of arguments for 'command' command"), false)
+	}
+	subcommand := strings.ToUpper(args[0])
+	switch subcommand {
+	case "COUNT":
+		return evalCOMMANDCOUNT()
+	default:
+		return Encode(fmt.Errorf("ERR unknown subcommand '%s'. Try COMMAND HELP", subcommand), false)
+	}
+}
+
+// evalCOMMANDCOUNT returns an number of commands supported by DiceDB
+func evalCOMMANDCOUNT() []byte {
+	return Encode(len(diceCommands), false)
```

Comment:
instead of calculating the length of the commands for every request, we can simply store it in a constant and return that. We can add a runtime check to verify that the constant's value does not get outdated.
```go
const (
    diceCommandsCount = len(diceCommands)
)

func init() {
    if len(diceCommands) != diceCommandsCount {
        panic("diceCommands slice has been modified")
    }
}
```